mergify_engine/tests/functional/base.py

```diff
@@ -614,12 +618,36 @@ async def create_pr(
         else:
             open(self.git.tmp + f"/test{self.pr_counter}", "wb").close()
             await self.git("add", f"test{self.pr_counter}")
-        await self.git("commit", "--no-edit", "-m", title)
+        args_commit = [
+            "commit",
+            "--no-edit",
+            "-m",
+            title,
+        ]
```

Comment:
Some comestic :smiley: 
```suggestion
        args_commit = ["commit", "--no-edit", "-m", title]
```